### ISSUE TITLE
Test EP Merge App in dev

### ISSUE THREAD

**Issue by dfitchett** [department-of-veterans-affairs/abd-vro#2449]
# User Story
As a user of the EP Merge App, I want to have EP Merge App deployed to the dev environment so that I can perform acceptance testing and verify connectivity between VRO apps and downstream services.

## Acceptance Criteria
Verify connectivity in dev to VRO apps and downstream:
- [x] Postgres / RDS
- [x] RabbitMQ
- [x] BIP API
- [x] ~BGS API~ Not possible to get to this point in the workflow for merge requests in dev. BGS will be tested separately.

## Related
Blocked by issues with dev deployments of VRO's RabbitMQ, BIP, BGS

**Comment by dfitchett** [comment]
Blocked in dev due to issue with deployment of rabbitmq. Also, need to change EP Merge App start hoppy clients in the background of startup task.

**Comment by meganhicks** [comment]
@amylai-nava - I think we should move this to next Sprint.

**Comment by amylai-va** [comment]
@meganhicks thanks for flagging. @dfitchett, are you still blocked in dev due to rabbitmq issues? if so, should we move this ticket out to the next sprint?

**Comment by amylai-va** [comment]
2/1: make a new ticket to track BGS testing
